lib/module-index-server/src/migrations/U0003__support_backups.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE modules
+    ADD COLUMN is_backup boolean DEFAULT false NOT NULL
```

Comment:
will likely end up with a type field as we'll probably end up with modules, backups, and just partial exports, but this is simpler for now...